samza-shell/src/main/bash/run-app.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
We already have a `run-job.sh` script right? How does this script relate to that? Is the intention to subsume functionality in `run-job.sh` into this, can't we remove `run-job.sh`? Or is that the end-goal?